templates/homepage.html

```diff
@@ -39,13 +39,18 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="#">MSC</a>
+          {% if session.logged_in %}
+          <a class="navbar-brand" href="/home">MSC</a>
+          {% else %}
+          <a class="navbar-brand" href="/">MSC</a>
+          {% endif %}
         </div>
         <div id="navbar" class="navbar-collapse collapse">
           <ul class="nav navbar-nav navbar-right">
+            {% if session.logged_in %}
```

Comment:
I should have done this before... dynamically render these based on whether you are logged in or not.